.ci/clusters/values-apache-default.yaml

```diff
@@ -0,0 +1,1019 @@
+#
```

Comment:
Can you avoid adding the entire default values file? You should just add the changes here. Please refer to other value files under the `.ci/clusters` directory.